examples/AdminClient/Program.cs

```diff
@@ -121,9 +121,18 @@ static List<AclBinding> ParseAclBindings(string[] args, bool many)
                 var operation = Enum.Parse<AclOperation>(args[baseArg + 5]);
                 var permissionType = Enum.Parse<AclPermissionType>(args[baseArg + 6]);
 
-                if (name == "") name = null;
-                if (principal == "") principal = null;
-                if (host == "") host = null;
+                if (name == "")
+                {
```

Comment:
this is fine, but i would usually put it on one line
`if (name == "") { name = null; }`